src/containers/tud/components/QuestionnaireForm.js

```diff
@@ -218,12 +224,16 @@ const QuestionnaireForm = ({
     validateAndSubmit
   } = pagedProps;
 
+  if (shouldReset) resetSurvey(setPage);
+
   const dispatch = useDispatch();
 
   const { schema, uiSchema } = formSchema;
 
-  const readingSchema = SecondaryFollowUpSchema.createSchema(READING);
-  const mediaUseSchema = SecondaryFollowUpSchema.createSchema(MEDIA_USE);
+  const activities :Object = trans(TranslationKeys.PRIMARY_ACTIVITIES, { returnObjects: true });
```

Comment:
The `:Object` annotation is no different than `:any`. I recommend either creating the type or removing `:Object`.